linked_list/linked_list.cpp

```diff
@@ -0,0 +1,426 @@
+/*
+
+* C++ Program to Implement Linked List
+
+*/
+
+#include<iostream>
+#include<cstdio>
+#include<cstdlib>
+
+using namespace std;
+
+/*
+
+* Node Declaration
+
+*/
+struct node
+{
+	int info;
+	struct node *next;
+}*start;
+
+
+/*
+
+* Class Declaration
+
+*/
+
+class single_llist
+
+{
+public:
+
+	node* create_node(int);
+	void insert_begin(int);
+	void insert_pos(int, int);
+	void insert_last(int);
+	void delete_pos(int);
+	void sort();
+	void search(int);
+	void update(int, int);
+	void reverse();
+	void display();
+
+	single_llist()
```

Comment:
use braces after the function name, not below the function name. For reference see other C++ codes.